From 16a6aa77fc5e326405f97da763bccd71b17381ab Mon Sep 17 00:00:00 2001
From: Bogdan Hamciuc <bogdan.hamciuc@nxp.com>
Date: Thu, 28 May 2020 10:03:50 +0300
Subject: [PATCH 127/269] s32g: Shift BL31 MMC offset

The current SD/MMC layout leaves BL31 in overlap with the U-Boot env, so
that a "saveenv" command will render the SD card unbootable.

Upstream-Status: Pending 

Signed-off-by: Bogdan Hamciuc <bogdan.hamciuc@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
Issue: ALB-5032
Issue: ALB-5033
---
 plat/s32g/include/platform_def.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plat/s32g/include/platform_def.h b/plat/s32g/include/platform_def.h
index d4bd3a24f..cbc7af44d 100644
--- a/plat/s32g/include/platform_def.h
+++ b/plat/s32g/include/platform_def.h
@@ -181,7 +181,7 @@
 #define BL31_BASE		(S32G_PMEM_START)
 #define BL31_LIMIT		(S32G_PMEM_END)
 
-#define BL31_MMC_OFFSET		(0xf2000)
+#define BL31_MMC_OFFSET		(0x1f2000)
 #define BL31_MMC_SIZE		(0x20000)
 
 /* FIXME value randomly chosen; should probably be revisited */
-- 
2.17.1

